http-api/src/lib.rs

```diff
@@ -46,18 +46,32 @@ pub struct Context {
 /// Run the HTTP API.
 pub async fn run(options: Options) {
     let paths = Paths::from_root(options.root).unwrap();
+    let storage = ReadOnly::open(&paths).expect("failed to read storage paths");
+    let peer_id = storage.peer_id().to_owned();
+
     let ctx = Context {
         paths,
         theme: options.theme,
     };
-    let api = path("v1")
-        .and(path("projects"))
+
+    let v1 = warp::path("v1");
+
+    let peer_id_route = path("peer")
+        .and(warp::get())
```

Comment:
We probably want `.and(path::end())`, otherwise this will also match `/v1/peer/lol`. Yeah, `warp` has great defaults /s

Reply:
yes, let me add.

Reply:
fixed in 7ed2c84